client/analytics/report/customers/table.js

```diff
@@ -104,7 +101,7 @@ export default class CustomersReportTable extends Component {
 		];
 	}
 
-	getCountryName( code ) {
+	getCountryName( code, countries ) {
```

Comment:
There's a default applied in `customers/index.js`, but a default was added in https://github.com/woocommerce/woocommerce-admin/pull/3581/commits/aa7b2b2618517d451fb9a8f06639198ba623864a to `getCountryName`

```js
const { countries = { countries: {} } } = dataEndpoints;
```

Reply:
On second thought, this destructuring default is not correct because the data in settings is an array. Its broken on master as well:

![Screen Shot 2020-01-20 at 10 23 44 AM](https://user-images.githubusercontent.com/1922453/72688769-5ca5cd80-3b6f-11ea-8c0f-dc1948439f55.png)

I will handle this separately.